src/controllers/api/account.ts

```diff
@@ -480,14 +481,14 @@ export class AccountController {
 		// 4. Check the token balance for Testnet account
 
 		// 1. Get logTo UserId from request body
-		const { username } = request.body;
+		const { name, primaryEmail } = request.body;
 
 		try {
 			// 2. Check if the customer exists
 			const customerEntity = response.locals.customer
 				? (response.locals.customer as CustomerEntity)
 				: // 2.1 Create customer
-					((await CustomerService.instance.create(username)) as CustomerEntity);
+					((await CustomerService.instance.create(name || primaryEmail, primaryEmail)) as CustomerEntity);
```

Comment:
Yes @ankurdotb. 

we have `UserEntity` which establishes the relationship between customer and logtoUserId.
This PR is replacing the uniqueness check with email instead of name.

Reply:
But the stripe email being same as the logto email / customer email will lead to a failure.